pkg/sql/internal.go

```diff
@@ -1007,9 +1004,7 @@ func (ie *InternalExecutor) execInternal(
 
 // ReleaseSchemaChangeJobRecords is to release the schema change job records.
 func (ie *InternalExecutor) releaseSchemaChangeJobRecords() {
```

Comment:
kill this function altogether. Look at the one place it was called. That thing can just invoke this function.